pkg/api/v1/atlasdatabaseuser_types_test.go

```diff
@@ -1,33 +1,22 @@
-package v1
+package v1 // nolint: dupl
```

Comment:
We did not need this before, now we do. Maybe the linter has a point and we should revisit our code?

Reply:
It complains about test cases, which are only data and don't make sense to be generalized. The generic part which actually tests and asserts data is already decoupled.